tools/cockpit.spec

```diff
@@ -238,6 +238,8 @@ BuildRequires:  python3-tox-current-env
 %endif
 
 cp %SOURCE1 tools/cockpit.pam
+cp %SOURCE1 tools/cockpit.pam-2fa
+echo 'auth required pam_oath.so usersfile=${HOME}/.pam_oath_usersfile window=20 digits=6' >> tools/cockpit.pam-2fa
```

Comment:
Can we just edit the `cockpit.pam` instead? We have it in the package. This should be set to optional (`nullok`) and then the admin can enforce it with manual override.

Reply:
@Vogtinator asked for a solution that a) does not require jeos-firstboot to mess with the content of pam config b) allows the cockpit package to upgrade pam config configuration. Having the admin to adjustments to the content means the package can not easily upgrade the content anymore.